docs/design/fabric-v2+/fpc-cc-invocation.puml

```diff
@@ -160,7 +177,7 @@ group invoke
 
     else Post-MVP: Validation
       Peer -> Peer : create proposal response <args, read/writeset, result=result_enc, ...>
-      Peer -> Peer : Replace endorsement signature & cert of peer with Sig_enclave & CC_Parameters.Certificate
+      Peer -> Peer : Replace endorsement signature & cert of peer with Sig_enclave & CC_Params.Certificate
```

Comment:
oh, I agree with you. somewhere in the UML I say Host_Params and just because it's shorter I changed to CC_Params as well. (Just had the next UML PR regarding beautifying in mind).